packages/gafl-webapp-service/package-lock.json

```diff
@@ -9,8 +9,6 @@
       "version": "1.43.0-rc.2",
       "license": "SEE LICENSE IN LICENSE",
       "dependencies": {
-        "@defra-fish/business-rules-lib": "1.43.0-rc.2",
-        "@defra-fish/connectors-lib": "1.43.0-rc.2",
```

Comment:
I don't think these should have been deleted - especially not with them still being in package.lock...